test/generate.cc

```diff
@@ -42,6 +42,7 @@
 #include "SimAttribute.h"
 #include "ModelTypes.h"
 
+static const pcu::PCU* globalPCU = nullptr;
```

Comment:
Issue here with messageHandler() and lines: 64, 68, and 300. Temporary fix is a global variable, but will be looked at soon.

Reply:
@cwsmith I don't think we remembered to discuss this on Friday. The Simmetrix message handler doesn't provide much of an option to get any data into it which requires a global variable.

Reply:
This is fine with me.